data/campaigns/Under_the_Burning_Suns/units/quenoth/Corrupted_Elf.cfg

```diff
@@ -74,3 +74,12 @@
         {SOUND:HIT_AND_MISS magic-dark.ogg magic-dark-miss.ogg -200}
     [/attack_anim]
 [/unit_type]
+
+[unit_type]
+    id=Defeated Corrupted Quenoth Elf
+    image=units/quenoth/corrupted-elf-defeated.png
+    [base_unit]
+        id=Corrupted Quenoth Elf
+    [/base_unit]
+    do_not_list=yes
```

Comment:
you could add hide_help=yes too

Reply:
@sevu How about merging as-is and adding hide_help, then cherry picking over to master? I'll do it if you don't want to.

Reply:
Doofus actually added it. The reason I didn't merge is because I don't know what to think of the weapon special renaming. (That shall not mean that it can't go in.)

Reply:
I agree with his comment: a custom weapon special should not have the same name as a standard ability. Let's merge this and if someone takes issue with the new name he chose, it can be discussed at that time.

Reply:
Alright